vscode/src/completions/providers/anthropic.ts

```diff
@@ -124,12 +124,34 @@ export class AnthropicProvider extends Provider {
         let remainingChars = this.promptChars - this.emptyPromptLength()
 
         for (const snippet of snippets) {
+            const formatRelationship = (
+                relationship:
+                    | {
+                          symbol: string
+                          relationship: 'typeDefinition' | 'implementation'
+                      }
+                    | undefined
+            ): string => {
```

Comment:
Can we reuse the type we already have?

```suggestion
            const formatRelationship = (
                relationship: SymbolContextSnippet['sourceSymbolAndRelationship']): string => {
```